app/adapters/signal_adapter/outbound.rb

```diff
@@ -35,6 +37,13 @@ def perform(text:, recipient:)
         http.request(req)
       end
       res.value # may raise exception
+    rescue Net::HTTPServerException => e
+      ErrorNotifier.report(SignalError.new, context: {
+                             code: e.response.code,
+                             message: e.response.message,
+                             headers: e.response.each_header(&:header),
```

Comment:
Does this work? There was [an error](https://sentry.io/organizations/tactilenews/issues/3381440601/?project=5686596&query=is%3Aunresolved) in Sentry… I think `e.response.to_h` might be an easier way to get a hash.

Reply:
when I attempted this locally it threw an error... I have since updated the code to fix the error

Reply:
Ah, it seems to be `to_hash` instead of `to_h` -- I always get confused about when to use `to_h` vs. `to_hash`